nservicebus/hosting/azure-functions-service-bus/custom-triggers_content_ASBFunctionsWorker_[,5].partial.md

```diff
@@ -0,0 +1 @@
+Custom trigger definitions for Azure Service Bus triggers using NServiceBus message handlers are currently not supported. Use the `ServiceBusTriggerAttribute` to configure the auto-generated Azure Service Bus trigger.
```

Comment:
is there a point prior to 5.1 where the trigger signature will match what's in 5.1? If so, I see no issue with retroactively supporting custom triggers back to where the signature changes

Reply:
The problem with the previous signature is that it has plenty of fancy parameters and the default value declaration must also exactly match, otherwise things will not work. Discussed it with @andreasohlund and decided to leave it out of scope for now. But technically yes we could show it with the previous trigger definition too.

Reply:
We decided to only show the new syntax